src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -93,63 +102,82 @@ All capabilities derived from invalid capabilities are themselves invalid, i.e.,
 All locations in registers or memory able to hold a capability are CLEN-bits wide including the valid tag bit.
 Those locations are referred as being _CLEN-bit_ in this specification.
 
-[#sec_cap_bounds]
-==== Bounds and Representability
+NOTE: When the valid tag is zero, the register or memory location may be used to store non-capability data.
+
+[#sec_cap_bounds_overview]
+==== Capability Bounds
 
 Capabilities encode memory bounds, i.e. the lowest and highest byte in memory that it is permitted to access when dereferenced for data memory access, or for instruction execution.
 
 Checking is on a byte-by-byte basis, so that it is possible for a memory access to be fully in-bounds, partially out-of-bounds or fully out-of-bounds.
 
-All partially or fully out-of-bounds accesses cause an exception.
+It is not permitted to make any partially or fully out-of-bounds memory accesses.
 
-Every capability has two bounds - _base_ and _top_.
-The _base_ is returned by the <<GCBASE>> instruction.
-There is no equivalent instruction for the _top_, but there is for the _length_ which is defined to be `length = top - base`, and is returned by <<GCLEN>>.
+Every capability has two memory address bounds - _base_ representing the lowest accessible byte, and _top_ representing one byte above the highest accessible byte.
 
 * The _base_ is XLEN-bits and is _inclusive_.
 * The _top_ is XLEN+1 bits and is _exclusive_.
 ** The extra bit is required to allow the bounds to include the top byte of memory.
+* The _length_ is XLEN-bits and is defined to be _top - base_, saturated to 2^XLEN^ -1
```

Comment:
I guess strictly speaking the length is also XLEN+1, just the accessor is saturating. Not sure it's worth calling out here

Reply:
ok fixed